### PR TITLE
update readme for stating/prod instructions

### PR DESCRIPTION
**Pull request by katsuraku**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,9 @@ Deployment is handled by CI when merging to master. The process uses tasks from
 CSS and JavaScript files are hashed and deployed to an S3 bucket behind an Akamai CDN (next-geebee.ft.com).
 
 Markup is deployed to a separate S3 bucket behind a Fastly CDN (handling url-management.ft.com).
+
+## Staging and production environments
```
Comment (Lucas-Gluchowski) [inline README.md]: this is not actually a staging environment but test.
Staging is a kind of production replica.

**Comment by FTgremlin** [comment]
giphy

File: README.md (hunk 1 above)
Comment (matthew-andrews) [inline README.md]: naming things is hard :sob:

Comment (katsuraku) [inline README.md]: I've changed staging to "staging" ... 🙇

**Pull request merged**